app/views/book_ratings/edit.html.slim

```diff
@@ -1,3 +1,5 @@
+p#notice
```

Comment:
You should have only one placeholder for notice and alerts in appllcation.layout, pls remove this